core/vm/opcodes.go

```diff
@@ -204,6 +207,12 @@ const (
 	LOG4
 )
 
+// 0xb0 range - control flow ops.
+const (
+	CALLF = 0xb0 + iota
+	RETF
```

Comment:
```suggestion
	CALLF OpCode = 0xb0
	RETF   OpCode = 0xb1
```
Externally defined, shouldn't be handed to `iota` to figure out the sequence

Reply:
Fixed.